test/integration/test_multi_factor_authentication.py

```diff
@@ -0,0 +1,155 @@
+"""
+Integration test for Bandwidth's Multi-Factor Authentication API
+"""
+
+import os
+import unittest
+from random import seed
+from random import randint
+
+import bandwidth
+from bandwidth.api import mfa_api
+from bandwidth.model.code_request import CodeRequest
+from bandwidth.model.messaging_code_response import MessagingCodeResponse
+from bandwidth.model.verify_code_request import VerifyCodeRequest
+from bandwidth.model.verify_code_response import VerifyCodeResponse
+from bandwidth.model.mfa_request_error import MfaRequestError
+from bandwidth.model.mfa_unauthorized_request_error import MfaUnauthorizedRequestError
+from bandwidth.model.mfa_forbidden_request_error import MfaForbiddenRequestError
+from bandwidth.model.voice_code_response import VoiceCodeResponse
+from bandwidth.exceptions import ApiException, UnauthorizedException, ForbiddenException
+
+# seed the random number generator for the verify request
+seed(randint(10, 500))
```

Comment:
Wondering why `seed` here and not in `setup`?

Reply:
Its currently not working correctly so I need to fix this